site/docs/pages/guides/tailwind.mdx

```diff
@@ -52,64 +52,64 @@ To override default colorscheme, you need to modify the following css variables:
 
 @layer base {
   :root {
-    --text-ock-inverse: theme(colors.gray.50);
-    --text-ock-foreground: theme(colors.gray.950);
-    --text-ock-foreground-muted: theme(colors.gray.600);
-    --text-ock-error: theme(colors.rose.600);
-    --text-ock-primary: theme(colors.indigo.600);
-    --text-ock-success: theme(colors.lime.600);
-    --text-ock-warning: theme(colors.orange.600);
-    --text-ock-disabled: theme(colors.gray.400);
-
-    --bg-ock-default: theme(colors.gray.50);
-    --bg-ock-default-hover: theme(colors.gray.200);
-    --bg-ock-default-active: theme(colors.gray.300);
-    --bg-ock-alternate: theme(colors.gray.200);
-    --bg-ock-alternate-hover: theme(colors.gray.300);
-    --bg-ock-alternate-active: theme(colors.gray.400);
-    --bg-ock-inverse: theme(colors.gray.100);
-    --bg-ock-inverse-hover: theme(colors.gray.200);
-    --bg-ock-inverse-active: theme(colors.gray.300);
-    --bg-ock-primary: theme(colors.indigo.600);
-    --bg-ock-primary-hover: theme(colors.indigo.700);
-    --bg-ock-primary-active: theme(colors.indigo.800);
-    --bg-ock-secondary: theme(colors.slate.200);
-    --bg-ock-secondary-hover: theme(colors.slate.300);
-    --bg-ock-secondary-active: theme(colors.slate.400);
-    --bg-ock-error: theme(colors.rose.600);
-    --bg-ock-warning: theme(colors.orange.600);
-    --bg-ock-success: theme(colors.lime.300);
-    --bg-ock-default-reverse: theme(colors.gray.950);
+    --ock-text-inverse: theme(colors.gray.50);
+    --ock-text-foreground: theme(colors.gray.950);
+    --ock-text-foreground-muted: theme(colors.gray.600);
+    --ock-text-error: theme(colors.rose.600);
+    --ock-text-primary: theme(colors.indigo.600);
+    --ock-text-success: theme(colors.lime.600);
+    --ock-text-warning: theme(colors.orange.600);
+    --ock-text-disabled: theme(colors.gray.400);
+
+    --ock-bg-default: theme(colors.gray.50);
+    --ock-bg-default-hover: theme(colors.gray.200);
+    --ock-bg-default-active: theme(colors.gray.300);
+    --ock-bg-alternate: theme(colors.gray.200);
+    --ock-bg-alternate-hover: theme(colors.gray.300);
+    --ock-bg-alternate-active: theme(colors.gray.400);
+    --ock-bg-inverse: theme(colors.gray.100);
+    --ock-bg-inverse-hover: theme(colors.gray.200);
+    --ock-bg-inverse-active: theme(colors.gray.300);
+    --ock-bg-primary: theme(colors.indigo.600);
+    --ock-bg-primary-hover: theme(colors.indigo.700);
+    --ock-bg-primary-active: theme(colors.indigo.800);
+    --ock-bg-secondary: theme(colors.slate.200);
+    --ock-bg-secondary-hover: theme(colors.slate.300);
+    --ock-bg-secondary-active: theme(colors.slate.400);
+    --ock-bg-error: theme(colors.rose.600);
+    --ock-bg-warning: theme(colors.orange.600);
+    --ock-bg-success: theme(colors.lime.300);
+    --ock-bg-default-reverse: theme(colors.gray.950);
```

Comment:
i like this, but could this be a breaking change if users are already overriding our variables?

Reply:
Yes, it will be a breaking change!  I've been working alongside Mind, I think it's a big improvement that we've wanted to do for some time now. With the new theming work it makes sense to apply these updates now!